src/components/CardContainer.jsx

```diff
@@ -0,0 +1,14 @@
+import Card from './Card';
+import Button from './Common/Button';
+
+const CardContainer = ({ cardInfo, handleCardClick, isEditMode, handleDelBtnClick }) => {
```

Comment:
```suggestion
const CardContainer = ({ cardInfo, onClick, isEditMode, onDelete }) => {
```
(마이너) 컴포넌트 이름으로 카드를 컨트롤하고 있는 컴포넌트임을 알 수 있어서 onClick이나 onDelete만 붙혀줘도 충분할 것 같아요~